modules/n1ql/pages/n1ql-language-reference/indexing-arrays.adoc

```diff
@@ -265,31 +265,32 @@ They can efficiently index and query recursive trees of arbitrary depth.
 
 The following examples use the {install-sample-buckets}[travel-sample] bucket that is shipped with Couchbase Server.
 
+[[example-1]]
```

Comment:
The ID on the example blocks themselves aren't really necessary unless yo'are planning to link directly to them from an xref.

Reply:
Sure. Some of them are indeed linked to directly, including this one. It was easier to give them all an ID than to work out which ones needed it. In fact I generally like to give all examples an explicit ID, because it's useful to be able to link to them from other documents.

Reply:
That make sense. Though I would encourage you to use a semantic ID rather that one that reinforces the document order. That way, if you do move them around, all you need to change are the textual labels...and you don't break any existing links.